src/q5/network-INGI1341/summary/network-INGI1341-summary.tex

```diff
@@ -472,91 +559,104 @@ \subsection{The network layer}
+    \item[Reliable]: Guarantees data arrival. (\textit{Hard to implement}.)
+    \item[Unreliable]: Imperfection:
         \begin{itemize}
-            \item Garantit seulement une majorité qui arrive (\textit{Souvent ce qui est écarté est du à l'overflow du buffer})
-            \item Peut duppliquer les packets sur le réseau
-            \item Peut délivrer un SDU différrents
-            \item A une taille limité des données
+            \item Only guarantees a majority arrives. (\textit{Usually, what's
+            left over is due to buffer overflow.})
+            \item Can duplicate packets on the network.
+            \item Can deliver different SDUs.
+            \item Has a limited data size.
         \end{itemize}
 \end{description}
- Ce service apporte deux nouvelles features par rapport au connectionless network layer service: error detection mechanism et multiplexing technique qui permet differencier les  application sur le host.
+
+This service brings two new features compared to connectionless network layer
+service: an error detection mechanism and a multiplexing technique that
+permits differenciating applications on the host.
 
 \subsubsection{Connection-oriented}
-Ici il y a trois phases : établir la connection, transferer les SDUs (\textit{la connection est
-bidirectionnel}) et fermer la connection.
+There are three phases to this: establishing a connection, transferring SDUs
+(\textit{the connection is bidirectional}) and closing the connection.
 
 \begin{itemize}
-    \item \textbf{Reliability} : Elle n'est garantie que lorsque la connection est terminé avec un ``gracefully``, sinon des pertes peuvent être observé
+    \item \textbf{Reliability}: Is only guaranteed when the connection
+    terminates with ``gracefully'', otherwise packet loss in possible.
 \end{itemize}
 
 \paragraph{\textbf{Connection}}
 
-\paragraph{Refus:} Soit de la part du destinataire, soit du provider.
-\begin{figure}[h]
+\paragraph{Refus:} Either by the receiver or the sender.
+\begin{figure}[ht]
     \centering
     \includegraphics[width=8cm]{refus.png}
-    \caption{Refus connection}
+    \caption{Refused connection.}
 \end{figure}
 
-\subparagraph{Three-way handshake:} est utilisé contre l'approche naive de la connexion ``aller-retour''.  Celle ci est définie par l'envoi d'un \textbf{CR}, répondu par \textbf{CA} qui répond \textbf{CA}.
-
-\begin{figure}[h]
+\subparagraph{Three-way handshake:} used against the naive approach that is back-and-forth connection. It requires three separate steps:
+\begin{itemize}
+     \item The first host sends the second host a SYN message with its own sequence number $x$, which the second host receives.
+     \item The second host replies with a SYN-ACK message with its own sequence number $y$ and acknowledgment number $x+1$, which the first host receives.
+     \item The first host replies with an ACK message with acknowledgment number $y+1$, which the second host receives and doesn't reply to.
```

Comment:
Not sure if it is mentioned afterwards: the seqnum of this ACK (which may actually contain the first data) is $x+1$, dans the seqnum of the next segment from the second host is $y+1$.
If it seems strange, it is: actually, seqnum don't express segment numbers but rather expected next byte number, and as the handshake has transmitted zero data, we still have the possibility to use it.

Reply:
Bruh.

Reply:
Correction still missing ;)